test.js

```diff
@@ -67,6 +67,26 @@ test('absolute path', () => {
   assert.equal(String(result), 'export const filepath = "/home/user/example.js";\n')
 })
 
+test('cwd (absolute)', () => {
+  const result = recma().use(recmaExportFilepath, { cwd: '/home/user/docs/' }).processSync({
+    cwd: '/home/user',
+    path: '/home/user/docs/example.md',
+    value: ''
+  })
+
+  assert.equal(String(result), 'export const filepath = "example.md";\n')
+})
+
+test('cwd (relative)', () => {
+  const result = recma().use(recmaExportFilepath, { cwd: 'docs' }).processSync({
+    cwd: '/home/user',
+    path: '/home/user/docs/example.md',
+    value: ''
+  })
+
+  assert.equal(String(result), 'export const filepath = "example.md";\n')
+})
+
```

Comment:
Could you also add a test for `cwd: '..'`? That’s were the difference between `path.join` and `path.relative` is important.